README.md

```diff
@@ -6,7 +6,7 @@
 [![Build Status](https://travis-ci.com/HumanCellAtlas/data-store.svg?branch=master)](https://travis-ci.com/HumanCellAtlas/data-store)
 [![codecov](https://codecov.io/gh/HumanCellAtlas/data-store/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/data-store)
 
-This repository maintains a prototype for the data storage system of the
+This repository maintains a data storage system of the
```

Comment:
"a data storage system" -> "the data storage system"

Reply:
Resolved